<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Reportbug-NG</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>69</y>
     <width>948</width>
     <height>590</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Query or Filter:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring>lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit" />
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="tableView" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid" >
        <bool>false</bool>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWebView" name="webView" >
       <property name="url" >
        <url>
         <string>http://reportbug-ng.alioth.debian.org/</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>splitter</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReportbug_NG" >
    <property name="title" >
     <string>Reportbug-NG</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuBugreport" >
    <property name="title" >
     <string>Bugreport</string>
    </property>
    <addaction name="actionNewBugreport" />
    <addaction name="actionAdditionalInfo" />
    <addaction name="actionCloseBugreport" />
    <addaction name="separator" />
    <addaction name="actionNewWnpp" />
   </widget>
   <addaction name="menuReportbug_NG" />
   <addaction name="menuBugreport" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>659</y>
     <width>948</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>30</y>
     <width>948</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewBugreport" />
   <addaction name="actionAdditionalInfo" />
   <addaction name="actionCloseBugreport" />
   <addaction name="actionNewWnpp" />
  </widget>
  <action name="actionNewBugreport" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/face-devilish.png</normaloff>:/face-devilish.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New Bugreport...</string>
   </property>
   <property name="statusTip" >
    <string>Create a new Bugreport for the current package.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAdditionalInfo" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/face-glasses.png</normaloff>:/face-glasses.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Additional Information...</string>
   </property>
   <property name="statusTip" >
    <string>Provide additional information for this Bugreport.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionCloseBugreport" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/face-angel.png</normaloff>:/face-angel.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Close Bugreport...</string>
   </property>
   <property name="statusTip" >
    <string>Close the current Bugreport.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionNewWnpp" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/face-smile-gearhead-female.png</normaloff>:/face-smile-gearhead-female.png</iconset>
   </property>
   <property name="text" >
    <string>New &amp;WNPP Bugreport...</string>
   </property>
   <property name="statusTip" >
    <string>Create a new WNPP Bugreport.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClearLineEdit" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/edit-clear-locationbar-rtl.png</normaloff>:/edit-clear-locationbar-rtl.png</iconset>
   </property>
   <property name="text" >
    <string>Clear</string>
   </property>
   <property name="statusTip" >
    <string>Clear.</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="icon" >
    <iconset resource="icons.qrc" >
     <normaloff>:/window-close.png</normaloff>:/window-close.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quit Reportbug-NG.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections/>
</ui>
